sdk/core/src/main/java/com/klaviyo/core/lifecycle/KlaviyoLifecycleMonitor.kt

```diff
@@ -70,5 +84,13 @@ internal object KlaviyoLifecycleMonitor : LifecycleMonitor, Application.Activity
         // Warning: onActivityDestroyed is unreliable, I'm not even going to try to broadcast it
     }
 
+    override fun onConfigurationChanged(newConfig: Configuration) {
+        broadcastEvent(ActivityEvent.ConfigurationChanged(newConfig))
+    }
+
+    override fun onLowMemory() {
```

Comment:
What was this intended for?

Reply:
I think it's part of the configuration change interface